fence-agents.spec.in

```diff
@@ -534,6 +535,21 @@ Fence agent for Nova compute nodes.
 %{_mandir}/man8/fence_evacuate.8*
 %endif
 
+%package cyberpower-ssh
+License: GPLv2+ and LGPLv2+
+Summary: Fence agent for CyberPower network PDUs
+%if 0%{?fedora} || 0%{?centos} > 7 || 0%{?rhel} > 7 || 0%{?suse_version}
```

Comment:
You can replace this if/else section with `Requires: openssh-clients`.

Reply:
Change pushed to MR.